flow/network.h

```diff
@@ -420,6 +419,8 @@ class INetwork {
 	// Provides a clock that advances at a similar rate on all connected endpoints
 	// FIXME: Return a fixed point Time class
 
+	virtual double timer() = 0;
```

Comment:
Can you explain the difference between now() and timer() in a comment here?

Also, does this mean we should transition existing callers of `::timer` to `INetwork::timer` ?

Reply:
If program behavior depends on the result of timer() it will cause non-determinism in simulation. Since I added behavior that depends on timer I had to change what it did in simulation. Unfortunately, I am using timer to measure the CPU time of a chunk of code with no waits, so with this implementation that will always result in zero seconds elapsing. I think I am going to change timer() in simulation to advance with every call to timer().